internal/pkg/describe/backend_service.go

```diff
@@ -178,7 +178,7 @@ func (d *BackendServiceDescriber) Describe() (HumanJSONStringer, error) {
 		if err != nil {
 			return nil, fmt.Errorf("retrieve rollback alarm names: %w", err)
 		}
-		if alarmNames != nil {
+		if len(alarmNames) != 0 {
```

Comment:
We should probably fix `AlarmDescriptions` too. We don't list everything unless in the future we need this functionality.